bin/check-all.sh

```diff
@@ -60,9 +60,9 @@ fi
 if [ -e "${base_dir}/gradlew" ]; then
   gradle_file="${base_dir}/gradlew"
 else
-  gradle_base_file=$(which gradle)
+  gradle_base_file=$(which gradle) && chk_gradle=$? || chk_gradle=$?
```

Comment:
Firstly, Thanks for the patch @renato2099 👍 
>> The `which` gradle failed

QQ: Did the command fail because `gradle` was not installed or `which` was not installed? For the former, wouldn't the `-f` in the current implementation would catch it? 

I wrote a small test to verify this:

Check with valid gradle path:
```
sh-3.2$ gradle_base_file=$(which gradle)
sh-3.2$ if [ -f "$gradle_base_file" ]; then    echo "gradle_base_file $gradle_base_file exists."; else    echo "gradle_base_file $gradle_base_file does not exist."; fi
gradle_base_file /usr/local/bin/gradle exists.
```

Check with invalid gradle path:
```
sh-3.2$ gradle_base_file=$(which gradle-notfound)
sh-3.2$ if [ -f "$gradle_base_file" ]; then    echo "gradle_base_file $gradle_base_file exists."; else    echo "gradle_base_file $gradle_base_file does not exist."; fi
gradle_base_file  does not exist.
```